e2e-testing/__tests__/transfers.test.ts

```diff
@@ -0,0 +1,137 @@
+import Long from 'long';
+import {
+  BECH32_PREFIX,
+  IndexerClient,
+  LocalWallet,
+  Network,
+  SocketClient,
+  SubaccountInfo,
+  ValidatorClient,
+} from '@dydxprotocol/v4-client-js';
+import {
+  Ordering,
+  SubaccountTable,
+  TransferColumns,
+  TransferFromDatabase,
+  TransferTable,
+} from '@dydxprotocol-indexer/postgres';
+import * as utils from './helpers/utils';
+import { DYDX_LOCAL_ADDRESS, DYDX_LOCAL_MNEMONIC } from './helpers/constants';
+
+describe('transfers', () => {
+  it('test deposit', async () => {
+    connectAndValidateSocketClient();
+    const wallet = await LocalWallet.fromMnemonic(DYDX_LOCAL_MNEMONIC, BECH32_PREFIX);
+
+    const validatorClient = await ValidatorClient.connect(Network.local().validatorConfig);
+    const indexerClient = new IndexerClient(Network.local().indexerConfig);
+
+    const subaccount = new SubaccountInfo(wallet, 0);
+
+    // Check USDC asset position before
+    let assetPosResp: any = await indexerClient.account.getSubaccountAssetPositions(
+      DYDX_LOCAL_ADDRESS,
+      0,
+    );
+    expect(assetPosResp).not.toBeNull();
+    const positions = assetPosResp.positions;
+    const usdcPositionSizeBefore = positions.length !== undefined && positions.length > 0 ? positions[0].size : '0';
+
+    // Deposit
+    await validatorClient.post.deposit(
+      subaccount,
+      0,
+      new Long(10_000_000),
+    );
+
+    await utils.sleep(5000);  // wait 5s for deposit to complete
```

Comment:
nit: add todo to use an API to make this less flaky

Reply:
done